tmt/steps/provision/hardware.fmf

```diff
@@ -0,0 +1,11 @@
+#examples:
+#beaker:
+#  translations:
+#    - requirement : boot.method
+#      template: '
+#      {% if (BEAKER_OPERATOR, BEAKER_VALUE) in [("==", "bios"), ("!=", "uefi")] %}
+#        {"key_value": {"_key": "NETBOOT_METHOD", "_value": "pxe"},}
+#      {% elif (BEAKER_OPERATOR, BEAKER_VALUE) in [("==", "uefi"), ("!=", "bios")] %}
+#        {"or": [{"key_value": {"_key": "NETBOOT_METHOD", "_value": "grub2"},}, {"key_value": {"_key": "NETBOOT_METHOD", "_value": "efigrub"},}]}
+#      {% endif %}
+#      '
```

Comment:
Spread the template into multiple lines, the output is supposed to be a YAML, so we can avoid `{` and `}` and make the structure be more visible and readable - we are not in the business of saving whitespace, we can afford to use spaces and empty lines to improve things for humans:

```
template: |
    {% if (BEAKER_OPERATOR, BEAKER_VALUE) in [("==", "bios"), ("!=", "uefi")] %}
    key_value:
        _key: NETBOOT_METHOD
        _value: pxe

    {% elif (BEAKER_OPERATOR, BEAKER_VALUE) in [("==", "uefi"), ("!=", "bios")] %}
    or:
      - key_value:
            _key: NETBOOT_METHOD
            _value: grub2
      - key_value:
            _key: NETBOOT_METHOD
            _value: efigrub
    {% endif %}
```